.idea/modules.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```

Comment:
we can probably consider adding all of/most of the `.idea` directory into our `.gitignore`, not a priority for this particular PR though :+1: